flexget/components/sites/sites/filelist_api.py

```diff
@@ -0,0 +1,180 @@
+from loguru import logger
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.requests import TimedLimiter, RequestException
+from flexget.utils.requests import Session as RequestSession
+
+logger = logger.bind(name='filelist')
+requests = RequestSession()
+
+
+class SearchFileList(object):
```

Comment:
No need to inherit from object

Reply:
it was _inherited_ from the old plugin 😄

Reply:
I know, not blaming you here, just an observation